src/components/Tooltip/index.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { ReactElement, RefObject } from 'react';
+import clsx from 'clsx';
+import ClickAwayListener from '../../utilities/ClickAwayListener';
+import TooltipContent from './components/TooltipContent';
+import { Props } from './types';
+
+
+function Tooltip({
+    className,
+    triggerEvent = 'hover',
+    content,
+    isOpen = false,
+    children,
+    ...otherProps
+}: Props): ReactElement {
+
+
+    const { ...otherChildProps } = children || {};
+    const ref: RefObject<HTMLDivElement> = React.useRef(null);
+    const [isTooltipOpen, setIsTooltipOpen] = React.useState(isOpen);
+    const [boundingRect, setBoundingRect] = React.useState();
+    const [anchorHeight, setAnchorHeight] = React.useState(0);
+    const [anchorWidth, setAnchorWidth] = React.useState(0);
+    const wrapperClasses = clsx(
+        'brew-Tooltip',
+        className,
+    );
+    const contentClasses = clsx(
+        'brew-Tooltip__wrapper',
+        { 'brew-Tooltip--open': isTooltipOpen },
+    );
+
+
+    React.useEffect(() => {
+        function onScroll(): void {
+            const targets = Array.from(ref?.current?.parentNode?.children ?? []);
+            if (typeof targets[0].getBoundingClientRect === 'function') {
+                setBoundingRect(targets[0].getBoundingClientRect());
+            }
```

Comment:
Same thing as above... a 0-width array fallback, but then assuming there is at least one element in the array.